app/src/features/apiClient/screens/PostmanImporterView/components/PostmanImporter/utils.ts

```diff
@@ -140,16 +141,16 @@ const createApiRecord = (item: any, parentCollectionId: string): Partial<RQAPI.A
     requestBody =
       formdata?.map((formData: { key: string; value: string }) => ({
         id: Date.now(),
-        key: formData.key,
-        value: formData.value,
+        key: formData?.key || "",
+        value: formData?.value || "",
```

Comment:
Can formdata key/value be undefined here? If yes, please update the types to handle that otherwise the check is unnecessary